packages/core/application/core-application-browser-internal/src/application_service.tsx

```diff
@@ -105,6 +107,7 @@ export class ApplicationService {
   private openInNewTab?: (url: string) => void;
   private redirectTo?: (url: string) => void;
   private overlayStart$ = new Subject<OverlayStart>();
+  private hasCustomBranding$: Observable<boolean> | undefined;
```

Comment:
Why do you need that stored as a property, given you're only retrieving then using it within the single `start` method? 

If it's about the `pipe->stop`, you can do it inline

```tsx
public async start({ customBranding }) {
   const piped = customBranding.hasCustomBranding$.pipe(takeUntil(this.stop$))
   <Foo hasCustomBranding$={piped}
}
hasCustomBranding$={this.hasCustomBranding$}